LayoutTests/media/media-source/media-managedmse-bufferedchange.html

```diff
@@ -35,6 +35,9 @@
             waitFor(video, 'error').then(failTest);
 
             run('sourceBuffer = source.addSourceBuffer(loader.type())');
+            testExpected('sourceBuffer.__proto__', ManagedSourceBuffer.prototype);
+            testExpected('sourceBuffer.onbufferedchange !== undefined', true);
```

Comment:
My comment was to add these checks in addition to your new event listener. Checking that the event listener gets called seemed useful too.

Reply:
indeed ; but right now there's another bug that makes it behave differently if the GPU process is turned on. I will fix it soon

Reply:
Oh, right. Fine.